lib/monad.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// id :: a -> a
+function id(x) {
+  return x;
+}
+
+// curry :: ((a, b) -> c) -> a -> b -> c
+function curry(f, ...args1) {
```

Comment:
Do you mean `common.curry` by classical `curry`? That function works the same as mine.
Also I've find that use `curry` only for testing so it can be deleted with no effect on library